app/assets/javascripts/backbone/views/loan_tab_view.coffee

```diff
@@ -0,0 +1,13 @@
+class MS.Views.LoanTabView extends Backbone.View
+
+  events: (params) ->
```

Comment:
events should be a hash, not a function. So remove `(params) ->` from here and your event handler should start working.